cli/script/command-executor.ts

```diff
@@ -166,14 +167,27 @@ function appAdd(command: cli.IAppAddCommand): Promise<void> {
         });
 }
 
+function getOwnerEmail(map: CollaboratorMap): string {
+    var ownerEmail: string = "";
+    if (map) {
+        Object.keys(map).forEach((email: string) => {
+            if ((<CollaboratorProperties>map[email]).permission === "Owner") {
```

Comment:
I think the SDK should have an equivalent of the service's Permissions class and "Owner" should be replaced with Permissions.Owner.  Because it's a "string enum" I don't think it can be in rest-definitions.d.ts.